lib/views/frontend/spree/checkout/payment/_stripe.html.erb

```diff
@@ -29,7 +29,7 @@
     };
 
     function createToken() {
-      stripe.createToken(card).then((result) => {
+      stripe.createToken(card, Spree.stripeAdditionalInfo).then((result) => {
```

Comment:
Forgot to pass the metadata through. We used to mix it in with the Stripe parameters https://github.com/Printavo/solidus_gateway/blob/96dcde25a19bce7adbceeba373f340d7a861af0c/lib/views/frontend/spree/checkout/payment/_stripe.html.erb#L60

In Stripe v3 it gets passed directly: https://stripe.com/docs/js/tokens_sources/create_token?type=cardElement#stripe_create_token-data